frontend/tests/utils/wmgUtils.ts

```diff
@@ -65,14 +67,17 @@ export async function goToWMG(page: Page, url?: string) {
   const targetUrl = url || `${TEST_URL}${ROUTES.WHERE_IS_MY_GENE}`;
   return await tryUntil(
     async () => {
-      await Promise.all([
-        page.waitForResponse(
-          (resp: { url: () => string | string[]; status: () => number }) =>
-            resp.url().includes("/wmg/v2/primary_filter_dimensions") &&
```

Comment:
waiting for this response to come back is actually not reliable, because in the app we fetch `primary_filter_dimensions` twice, where the second call keeps the heat map in loading state

The double fetching is caused by snapshotId changing from `null` to a number - but not a bug to fix today since it's been in the app since inception)